Makefile

```diff
@@ -3,7 +3,7 @@ SHELL = /bin/sh
 DOCKER ?= $(shell which docker)
 DOCKER_REPOSITORY := graze/composer
 
-BATS ?= $(shell which bats)
+BATS := $(shell which bats)
```

Comment:
The assignment operator here was changed due to a difference in the behaviour of `ifdef` for reasons I have not yet worked out. More information here: http://stackoverflow.com/questions/35684686/why-does-conditional-variable-assignment-set-a-value-but-standard-assignment-doe